site/themes/arangodb-docs-theme/layouts/partials/shortcodes/examples/example.html

```diff
@@ -0,0 +1,54 @@
+{{ $context := .context }}
+{{ $content := .content }}
+{{ $language := .language }}
+
+{{ $metadata := findRE `(?ms)---.*---` $content }}
```

Comment:
This uses greedy matching and doesn't check whether `---` is at the beginning of a line and the only thing on the line